### PR TITLE
Revamp Expert Identity Provider Wizard

### PR DESCRIPTION
**Pull request by thivi**
- The default values were removed from the name and description fields.
- Hints were removed in favor of helpful placeholders.
- The certificate section was removed.
- A new form was created instead of resuing the same form used in the edit view.

image

### CODE REVIEW

File: apps/console/src/features/identity-providers/components/wizards/steps/identity-create-steps/general-settings.tsx [hunk 1]
```diff
@@ -47,16 +52,98 @@ export const GeneralSettings: FunctionComponent<GeneralSettingsWizardFormPropsIn
         [ "data-testid" ]: testId
     } = props;
 
+    const [ isNameValid, setIsNameValid ] = useState(false);
```
Comment (DimalChandrasiri) [inline apps/console/src/features/identity-providers/components/wizards/steps/identity-create-steps/general-settings.tsx:55]: Shall we add the state type as well here?

File: modules/i18n/src/translations/en-US/portals/dev-portal.ts [hunk 2]
```diff
@@ -1904,16 +1904,17 @@ export const devPortal: DevPortalNS = {
...
-                        placeholder: "https://example.com/image01"
+                        placeholder: "E.g. https://example.com/image.png"
                     },
                     name: {
                         hint: "Enter a unique name for this identity provider.",
                         label: "Identity Provider Name",
+                        placeholder: "Enter a name for the identity provider.",
                         validations: {
                             duplicate: "An identity provider already exists with this name",
                             empty: "Identity Provider name is required"
```
Comment (DimalChandrasiri) [inline modules/i18n/src/translations/en-US/portals/dev-portal.ts:1920]: Shouldn't these phrase changes needs to be reflected in other languages as well?

**Review by DimalChandrasiri — CHANGES_REQUESTED** [review]
Please resolve the comments.

File: apps/console/src/features/identity-providers/components/wizards/steps/identity-create-steps/general-settings.tsx (hunk 1 above)
Comment (thivi) [reply apps/console/src/features/identity-providers/components/wizards/steps/identity-create-steps/general-settings.tsx]: Resolved by 90dd7d8d

File: modules/i18n/src/translations/en-US/portals/dev-portal.ts (hunk 2 above)
Comment (thivi) [reply modules/i18n/src/translations/en-US/portals/dev-portal.ts:1920]: We currently support only English. I have cteated an issue to track localization. https://github.com/wso2/product-is/issues/9562

**Review by DimalChandrasiri — APPROVED** [review]

**Pull request merged**